src/index.html

```diff
@@ -78,22 +78,39 @@ <h4><a href="https://pub.dartlang.org/">Dart packages</a></h4>
-          <h2 class="core-goals__first">Provide a solid foundation of libraries and tools</h2>
-          <p>A programming language is nothing without its core libraries and tools. We call it <em>batteries included</em>. Dart’s batteries have been powering very large apps for years now.</p>
-          <h2>Make common programming tasks easy</h2>
-          <p>Application programming comes with a set of common problems and common errors. Dart is designed to make those problems easier to tackle, and errors easier to catch. This is why we have <code>async/await</code>, generators, string interpolation, earlier error detection and much more.</p>
-          <h2>Don’t surprise the programmer</h2>
-          <p>There should be a direct correspondence between what you type and what’s going to happen. Magic (automatic type coercion, hoisting, “helper” globals, &hellip;) doesn’t mix well with large apps.</p>
-          <h2>Be the stable, pragmatic solution for real apps</h2>
-          <p>Dart might seem boring to some. We prefer the terms <em>productive</em> and <em>stable</em>. We work closely with our core customers&mdash;the developers who build large applications with Dart&mdash;to make sure we’re here for the long run.</p>
+          <h2 class="core-goals__first">Productive</h2>
+          <p>Syntax must be clear and concise, tooling simple, and dev cycles near-instant and on-device.</p>
+          <p>Dart increases developer velocity because it has a clear, succinct syntax and is able to run on a 
+            VM with a JIT compiler. The latter allows for <a href='https://flutter.io/hot-reload/'>stateful hot reload</a> 
+            during mobile development, resulting in super fast dev cycles, where you can edit code, compile and replace 
+            in the running app on the device.</p>
+          <h2>Fast</h2>
+          <p>Runtime performance and startup must be great and predictable even on small mobile devices.</p>
+          <p>With its ability to efficiently compile to native code ahead of time, Dart provides predictable, 
+            high performance and fast startup on mobile devices.</p>
+          <h2>Portable</h2>
+          <p>Client developers have to think about three platforms today: iOS, Android, and Web. The language needs to work 
+            well on all of them.</p>
+          <p>Dart supports compilation to native code (ARM, x86, etc.) for fast mobile performance as well as transpilation 
+            to efficient JavaScript for the web.</p>
+          <h2>Approachable</h2>
+          <p>The language can’t stray too far from the familiar if it wishes to be relevant for millions of developers.</p>
+          <p>Dart is approachable to many existing developers, thanks to its unsurprising object-oriented aspects and syntax 
+            that — according to our users — allows any C++, C#, Objective-C, or Java developer to be productive in a matter 
+            of days.</p>
+          <h2>Reactive</h2>
```

Comment:
I wouldn't say "Dart is reactive", which is how the intro sets this up ("Dart ~is~ **helps you be** productive", "Dart is fast", "Dart is portable", "Dart is approachable", "Dart ~is~ **supports** reactive **programming**"). Some of the other adjectives don't work as well, but "reactive" stands out to me. Maybe "Flexible"? But it seems like you want "Reactive" in a big heading?

I don't have a good suggestion for how to fix this, and I suppose I could live with it if we can't come up with something better.

Reply:
I'd like to try to reword the intro section so this is less about what Dart strictly speaking "is" and more about the key benefits you get from using Dart.